datafusion/functions/src/datetime/to_char.rs

```diff
@@ -172,6 +171,16 @@ fn _to_char_scalar(
     let data_type = &expression.data_type();
     let is_scalar_expression = matches!(&expression, ColumnarValue::Scalar(_));
     let array = expression.into_array(1)?;
+
+    if format.is_none() {
+        if is_scalar_expression {
+            return Ok(ColumnarValue::Scalar(ScalarValue::Utf8(None)));
+        } else {
+            let result: Vec<Option<String>> = vec![None; array.len()];
+            return Ok(ColumnarValue::Array(Arc::new(StringArray::from(result))));
```

Comment:
The performance of this likely doesn't matter, but you can do the same thing more efficiently via `new_null_array`:

https://docs.rs/arrow/latest/arrow/array/fn.new_null_array.html

Reply:
Done.